tests/contrib/scrapy/test_pipeline_cerberus.py

```diff
@@ -135,4 +143,58 @@ def test_validation_from_url(mocker):
     pipe.process_item(test_item, None)
 
     assert "spidermon/validation/items/errors" in pipe.stats.stats.get_stats()
-    assert pipe.stats.stats.get_stats().get("spidermon/validation/validators/testitem/cerberus", True)
+    assert pipe.stats.stats.get_stats().get(
+        "spidermon/validation/validators/testitem/cerberus", True
+    )
+
+
+@pytest.mark.skipif(
```

Comment:
Are you sure about this?

Reply:
`Requires: Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*`
Source: [PyPI](https://pypi.org/project/pytest-mock/)

Since Python2 goes out of force soon, hence I think this is a good point that I picked from other tests that I read online and in JSONSchema too.